src/main/java/org/kilocraft/essentials/util/monitor/NetherInfo.java

```diff
@@ -0,0 +1,38 @@
+package org.kilocraft.essentials.util.monitor;
+
+import net.minecraft.server.world.ServerWorld;
+import org.kilocraft.essentials.api.world.MonitorableWorld;
+
+public class NetherInfo implements MonitorableWorld {
```

Comment:
NetherInfo, OverworldInfo and TheEndInfo are all functionally redundant. 

MonitorableWorld should really be a Mixin into ServerWorld (So it will work on any ServerWorld instance including other dimensions that aren't in the vanilla game). 

Really the interface is useless and a WorldMonitorHelper to just get all those fields you want from the world is probably the least messy option or just directly call your methods from the ServerWorld where you use them.